sys/net/application_layer/nanocoap/sock.c

```diff
@@ -817,3 +817,38 @@ int nanocoap_server(sock_udp_ep_t *local, uint8_t *buf, size_t bufsize)
 
     return 0;
 }
+
+static kernel_pid_t _coap_server_pid;
+static void *_nanocoap_server_thread(void *local)
+{
+    static msg_t _msg_queue[1 << CONFIG_NANOCOAP_SERVER_QUEUE_EXP];
+    static uint8_t buf[CONFIG_NANOCOAP_SERVER_BUF_SIZE];
+
+    msg_init_queue(_msg_queue, ARRAY_SIZE(_msg_queue));
```

Comment:
tbh I'm not sure if this is needed as nanoCoAP uses the sock API which afaik uses mbox